### PR TITLE
remove case spec/case edit views and all dependencies

### PR DESCRIPTION
**Pull request by czue**
this was an idea that @dannyroberts had for making domain specific edit urls 3 years ago. there are actually 0 usages of this data model on prod and we can get rid of a couch view in the process.

### CODE REVIEW

**Comment by czue** [comment]
@emord or whoever

File: corehq/apps/cloudcare/urls.py [hunk 1]
```diff
@@ -8,13 +8,7 @@
     url(r'^(?P<urlPath>.*)$', 'cloudcare_main', name='cloudcare_main'),
 )
 
-cases_urls = patterns('corehq.apps.cloudcare.views',
-    url(r'^view/(?P<case_id>[\w-]*)/$', 'view_case', name='cloudcare_view_case'),
-    url(r'^create/$', 'view_case', {'case_id': None}, name='cloudcare_create_case'),
-)
-
 api_urls = patterns('corehq.apps.cloudcare.views',
-    url(r'^groups/(?P<user_id>[\w-]*)/$', 'get_groups', name='cloudcare_get_groups'),
```
Comment (czue) [inline corehq/apps/cloudcare/urls.py]: I sanity checked that these weren't being used on the proxy just to be sure:

czue@hqproxy0:~$ grep /cloudcare/cases/  /home/cchq/www/production/log/cchq-apache.log | wc
      0       0       0
czue@hqproxy0:~$ grep /cloudcare/api/groups/  /home/cchq/www/production/log/cchq-apache.log | wc
      0       0       0

(and to verify the command works)
czue@hqproxy0:~$ grep /cloudcare/apps/  /home/cchq/www/production/log/cchq-apache.log | wc
 457248 9614334 150826846

**Comment by emord** [comment]
4 additions and 565 deletions. :+1:

**Comment by czue** [comment]
whoever is deploying should kick off a preindex now @TylerSheffels @nickpell @esoergel

**Comment by czue** [comment]
or maybe after you merge https://github.com/dimagi/commcare-hq/pull/6748 too  ;)

**Pull request merged**